src/backend/tools.ts

```diff
@@ -141,6 +141,15 @@ export const DXVK = {
       return true
     }
 
+    if (gameSettings.wineVersion.bin.includes('toolkit')) {
```

Comment:
```suggestion
    if (gameSettings.wineVersion.type === 'toolkit') {
```

I think it's faster than includes

Reply:
maybe we can do this check at the beginning of the function? we don't need to check the prefix if we are going to skip this anyway